collector.ts

```diff
@@ -3,9 +3,7 @@
 export const testableUrls = new Map<string, string[]>();
 
 export function addTests(test: string, urls: string[]): void {
```

Comment:
I also tried many variations of `--pure` and `--define`. I wasn't able to drop this `addTests` helper